model/span.go

```diff
@@ -96,6 +96,16 @@ func (s *Span) NormalizeTimestamps() {
 	}
 }
 
+// FlattenTags combines span tags, process tags, and log fields into one KeyValues collection
```

Comment:
This is used just once and the semantic value of mixing tags with log fields is highly dubious. I would avoid having this in the model.

Reply:
The only reason I extracted this as a public method was a `TODO` by the existing function suggesting this fix. Can easily remove.